pkg/repro/repro.go

```diff
@@ -133,7 +135,9 @@ func Run(crashLog []byte, cfg *mgrconfig.Config, vmPool *vm.Pool, vmIndexes []in
 
 	res, err := ctx.repro(entries, crashStart)
 	if res != nil {
+		ctx.reproLog(2, "repro crashed as:\n%s", string(ctx.report))
```

Comment:
Do we need full report on log level 2? It can be hundreds of lines and will appear in manager output.
I think we are mostly interested in desc, which we use for bug identification and already print on log level 2.
If you think we need it, make it level 4 or something.